specification/eventgrid/data-plane/readme.md

```diff
@@ -333,13 +333,14 @@ input-file:
   - $(this-folder)/Microsoft.ResourceNotifications/stable/2018-01-01/common.json
   - $(this-folder)/Microsoft.ResourceNotifications/stable/2018-01-01/HealthResources.json
   - $(this-folder)/Microsoft.ResourceNotifications/stable/2018-01-01/Resources.json
+  - $(this-folder)/Microsoft.AVS/preview/2023-09-01-preview/PrivateCloud.json
```

Comment:
Please use the existing stable/2018-01-01 pattern. Also I think this entry needs to be added in the sections above as well.